offchain-modules/packages/x/src/db/ckb.ts

```diff
@@ -87,4 +87,41 @@ export class CkbDb {
     const dbRecords = records.map((r) => btcUnlockRepo.create(r));
     await btcUnlockRepo.save(dbRecords);
   }
+
+  async removeUnconfirmedCkbBurn(currentBlockHeight: number, confirmNumber: number): Promise<DeleteResult> {
```

Comment:
Why do you use two params here instead of just one `confirmedBlockHeight`?

Reply:
fixed